### ISSUE TITLE
Final Meeting

### ISSUE THREAD

**Issue by db626** [FACG5/Twig#366]
Dear Group,

Thanks for the meeting today. I'll talk to the new GSG manager this week and we'll see how to proceed. Of course the original plan was to get this bit of Twig running then ask for outside funding when we prove that crowdsourcing medical translation works. As it stands now I'll have to look at funding finishing up myself. We'll see how that goes- the new GSG manager could tell me that continuing could be $90, $900, $9000, $90,000 etc.

As the person who runs Twig, it's my job to be very exacting about the product we offer, for a number of reasons.

Anyway, I wanted to write a note not as a Product Owner, but just as a person wishing you the best on your graduation from GSG.

As I told you in our original meeting I was raised in the Middle East when I was very young. Specifically, my father was a United Nations 'peacekeeper' located there from 1979-81. Many nations were included in the group- Americans, Russians, etc. Peacekeepers brought their families and rotated countries. I was born in 1978, so I don't remember much of it. My father, a veteran of Vietnam was a rather devout athiest by the time I was born... I do remember thinking at a very young age that 'nations' and national armies were a thing of the past- and probably rather naievely- that international cooperation were going to do the heroic actions in future. When I was small my mother taught me basic words in Arabic. My mother and I left the Middle East when I was 3, there were bombings of US facilities in Israel/Gaza at the time, a new war in Lebanon, and the assasination of Sadat in Egypt (these events are positively prehistoric to you all, I realize, and *much* history has happened since.

While I was growing up my mother told me exciting stories about the Middle East- about the stone where Muhammad ascended into heaven, about Anwar Sadat's wife and all the reforms she wanted, about the food she ate and wonderful people she met. She took thousands of photos. Our walls were covered with items she bought in markets there (so much so that I used to call our living room 'the temple of doom'). She also used to say that education is the only thing you can buy that can't be stolen from you- a bit of an old-fashioned idea.

If I have thus far written anything offensive, or culturally tone-deaf, I appologize.

What I wanted to say is this: I decided at some point in my own life that heroes probably won't wear any uniforms at all- not national ones *or* international ones. They'll probably be a series of individuals with skills and keyboards, doing good- geeks. 

This is the spirit I have tried to follow with Twig.

On your graduation from GSG I first wish you four that your skills get you jobs (it's hard to change the world if you're not eating and living somewhere). If we don't work with each other again I wish you the best in employment for yourselves (first), thus a better life for your families, and in turn your community. If you work on the occasional tech project for social good I wouldn't complain either.

Congratulations for getting through the GSG program!

Geeks are the future.

Shukran.

Bradley.

P.S. I'm attaching some photos from an evening in Gaza, circa 1980. The little blonde person is me. 

![083-gaza](https://user-images.githubusercontent.com/44851386/49944011-41cf4700-fee1-11e8-9c77-6d062e8c237b.jpg)
![085-gaza](https://user-images.githubusercontent.com/44851386/49944012-41cf4700-fee1-11e8-80e4-725c93bb3949.jpg)

![327-oct1980 box5](https://user-images.githubusercontent.com/44851386/49944013-4267dd80-fee1-11e8-8f7d-f404c0e2eb25.jpg)

**Comment by db626** [comment]
![140-unknown box3aaa](https://user-images.githubusercontent.com/44851386/49944306-041eee00-fee2-11e8-90ed-a70572d3d2b9.jpg)
![176-unknown box3aaa](https://user-images.githubusercontent.com/44851386/49944307-04b78480-fee2-11e8-8ecf-d6fa52fd6669.jpg)
![327-oct1980 box5aaa](https://user-images.githubusercontent.com/44851386/49944308-04b78480-fee2-11e8-9b3e-36093b1b9658.jpg)

**Comment by FarahZaqout** [comment]
@ali-7 @lubnaabd @HemaSAli check out our P.O when she was a little angel.

That restaurant on the beach I think is still open. It looks like Al Ma'moura.

**Comment by ali-7** [comment]
Lovely photos :heart: Bradley :heart: , we really enjoyed working with you and working on Twig, thank you for this opportunity, and for allowing us to know you and know the passion you have for helping people and make good for the world.

Thank you :heart:

**Comment by db626** [comment]
Yes, Farah, this sounds like the right name of the place- Al Ma'moura... mom would bring out the slide carousels repeatedly and talk guests through place names (we've even got a file labeled 'end of the world'- where armageddon is supposed to start- go figure).

I digitized the slides/negatives/film some years ago, but they mostly just sat collecting digital dust. I've never really seen them this large- on a computer screen- you can really pick out some small details in even the average photos.

Thanks for the lovely thoughts, I've read them all twice. 

I hope you enjoy this week's vacation and will work on a presentation this weekend for the next GSG manager.

Sending more photos, partly out of egotism, partly because although I've had my own international experiences (Italy), my worldview was probably largely decided in Gaza, before I was aware of it. I promise no more photos will follow.

@ali-7 @lubnaabd @HemaSAli 

![066-gaza july4](https://user-images.githubusercontent.com/44851386/50010211-b329fb00-ffb0-11e8-8cbc-255fe4ef8b14.jpg)
![073-gaza july4](https://user-images.githubusercontent.com/44851386/50010212-b3c29180-ffb0-11e8-806b-d20df6ea3ce3.jpg)
![074-july4](https://user-images.githubusercontent.com/44851386/50010213-b3c29180-ffb0-11e8-863f-b63f48610573.jpg)
![075-july4](https://user-images.githubusercontent.com/44851386/50010214-b3c29180-ffb0-11e8-8114-857fb0bf17b3.jpg)
![076-july4](https://user-images.githubusercontent.com/44851386/50010215-b3c29180-ffb0-11e8-9155-4a0d4677f690.jpg)
